shell/browser/file_system_access/file_system_access_permission_context.cc

```diff
@@ -652,11 +655,11 @@ void FileSystemAccessPermissionContext::DidCheckPathAgainstBlocklist(
         "file-system-access-restricted", details,
         base::BindRepeating(
```

Comment:
Not new to this PR, but asking anyway to make sure  I'm not misunderstanding something.

Since `OnRestrictedPathResult()` -> `RunRestrictedPathCallback()`  consumes the callback being used, seems like this would only need to be called once, and so we'd want to use `base::BindOnce()` here instead of `base::BindRepeating()`?